src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -956,6 +960,34 @@ using namespace ngraph::runtime;
         writer += "}\n\n";
     }
 
+    // TODO: Cleanup and make this a utility function
+    file_util::make_directory(s_output_dir);
```

Comment:
What is cleaning up these files?

Reply:
This is existing code which was moved to place it closer to the codegen path. 
`StaticInitializers() { ngraph::file_util::remove_directory(s_output_dir); }`